words/predicates.txt

```diff
@@ -651,6 +739,7 @@ nifty
 night
 nimble
 nine
+ninth
 nippy
```

Comment:
I've always assumed this was a term for cold weather because it leads to people's nipples being visible, so maybe it's not a great predicate?